README.md

```diff
@@ -339,106 +339,348 @@ alpaca.deleteFromWatchlist('xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', "AAPL").then(
-
-#### Latest trade
-
 ```ts
 getLatestTrade(
   symbol: string,
-  config?: any
-) => Promise<AlpacaTrade>
+  config?: any,
+): Promise<AlpacaTrade>;
 ```
-Or if you want to get the latest trades for multiple symbols
 ```ts
 getLatestTrades(
   symbols: Array<string>,
-  config?: any
-): Promise<Map<string, AlpacaTrade>>
+  config?: any,
+): Promise<Map<string, AlpacaTrade>>;
 ```
-###### example
-```js
+##### Example
+```ts
 const trade = await alpaca.getLatestTrade('AAPL');
-
 console.log(trade);
 ```
+###### Output:
+```python
+{
+  Timestamp: '2022-04-14T13:54:24.907840256Z',
+  Exchange: 'P',
+  Price: 169.33,
+  Size: 100,
+  Conditions: [ '@' ],
+  ID: 12272,
+  Tape: 'C'
+}
+```
 
-#### Latest quote
+#### Quotes
 ```ts
-getLatestQuote(
+getQuotesV2(
   symbol: string,
-  config?: any
-) => Promise<AlpacaQuote>
+  options: GetQuotesParams,
+  config?: any,
+): AsyncGenerator<AlpacaQuote, void, unknown>;
+```
+```ts
+getMultiQuotesV2(
+  symbols: Array<string>,
+  options: GetQuotesParams,
+  config?: any,
+): Promise<Map<string, any[]>>;
+```
+```ts
+getMultiQuotesAsyncV2(
+  symbols: Array<string>,
+  options: GetQuotesParams,
+  config?: any,
+): AsyncGenerator<AlpacaQuote, void, unknown>;
+```
+```ts
+ getLatestQuote(
+   symbol: string,
+   config?: any,
+): Promise<AlpacaQuote>;
 ```
-Or if you want to get the latest quotes for multiple symbols
 ```ts
 getLatestQuotes(
+   symbols: Array<string>,
+   config?: any,
+): Promise<Map<string, AlpacaQuote>>;
+```
+##### Example
+```ts
+const trades = await alpaca.getMultiTradesV2(["PFE", "SPY"], {
+  start: "2022-04-18T08:30:00Z",
+  end: "2022-04-18T08:31:00Z",
+  limit: 2,
+});
+console.log(trades);
+```
+###### Output:
+```ruby
+{
+  'PFE' => [
+    {
+      Timestamp: '2022-04-18T08:30:59.988642304Z',
+      Exchange: 'P',
+      Price: 53.25,
+      Size: 5,
+      Conditions: [Array],
+      ID: 52983525028174,
+      Tape: 'A',
+      Symbol: 'PFE'
+    }
+  ],
+  'SPY' => [
+    {
+      Timestamp: '2022-04-18T08:30:00.066013952Z',
+      Exchange: 'P',
+      Price: 436.39,
+      Size: 1,
+      Conditions: [Array],
+      ID: 52983525028949,
+      Tape: 'B',
+      Symbol: 'SPY'
+    }
+  ]
+}
+```
+
+##### Bars
+```ts
+ getBarsV2(
+   symbol: string,
+   options: GetBarsParams, 
+   config?: any,
+ ): AsyncGenerator<AlpacaBar, void, unknown>;
+```
+```ts
+getMultiBarsV2(
   symbols: Array<string>,
-  config?: any
-): Promise<Map<string, AlpacaQuote>>
+  options: GetBarsParams,
+  config?: any,
+): Promise<Map<string, any[]>>;
+```
+```ts
+ getMultiBarsAsyncV2(
+   symbols: Array<string>,
+   options: GetBarsParams,
+   config?: any,
+ ): AsyncGenerator<AlpacaBar, void, unknown>;
+```
+```ts
+ getLatestBar(
+  symbol: string,
+  config?: any,
+): Promise<AlpacaBar>;
+```
+```ts
+getLatestBars(
+  symbols: Array<string>,
+  config?: any,
+): Promise<Map<string, AlpacaBar>>;
 ```
-###### example
```

Comment:
No Example for Bars?

Reply:
The example file I linked under contains a case for bars, I don't think it is necessary to add examples in the readme for all the different types, but if you want I can add there as well.

Reply:
I think it makes sense to have it since we have examples for the rest of the sections here. 

Also what link to the file are you referring to? I don't see a link here in the bars section to an example file? :thinking:

Reply:
It is the end of this section (~L567). Anyways I'll add an example there as well.